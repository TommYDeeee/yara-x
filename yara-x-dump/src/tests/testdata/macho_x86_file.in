field1: 123
field2: "test"
field3: "test\ntest"
segments {
    nested1: 456
    nested2: 789
    timestamp: 123456789
}
segments {
    nested1: 100000
    nested2: 200000
    timestamp: 999999999
}
optional {
    onested1: 123
    onested2: 456
    map_string_string {
      key: "foo\nfoo"
      value: "bar\nbar"
    }
}
