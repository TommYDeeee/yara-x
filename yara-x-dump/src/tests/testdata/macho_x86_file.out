field1: 0x7b
field2: "test"
field3: "test\ntest"
segments:
  - nested1: 456
    nested2: 0x315
    timestamp: 123456789 # 1973-11-29 21:33:09 UTC
  - nested1: 100000
    nested2: 0x30d40
    timestamp: 999999999 # 2001-09-09 01:46:39 UTC
optional:
onested1: 123
    onested2: 0x1c8
    map_string_string:
        "foo\nfoo": "bar\nbar"
